### ISSUE TITLE
can we launch wsa app in default fullscreen mode?

### ISSUE THREAD

**Issue by MaxLEAF3824** [microsoft/WSA#133]
### Is your feature request related to a problem? Please describe

Now, we can make a wsa app fullscreen by pressing `F11` on keyboard after the app is launched. Can we launch a app in default fullscreen mode(without pressing the `F11`)

### Describe the solution you'd like

I guess adding a **shell command option** to WsaClient.exe will be nice, like:  
`wsaclient /launch wsa://com.android.settings/com.android.settings.Settings --fullscreen `  
or add an fullscreen option in **Windows Subsystem for Android™️ Settings. > Compatibility**

### Please specify the version of Windows Subsystem for Android

2209.40000.26.0

**Comment by bansalsushant** [comment]
Can you elaborate on why do you want this feature? Which app will benefit from this?

**Comment by yonggamer** [comment]
I like add an fullscreen option in Windows Subsystem for Android Settings > Compatibility

**Comment by yonggamer** [comment]
#122

**Comment by yonggamer** [comment]
just like google play games for pc

**Comment by nieubank** [comment]
Does your app on Google Play Games for PC use immersive mode or otherwise call some kind of API to put it into fullscreen on other freeform Android devices?

**Comment by yonggamer** [comment]
I dont know but i see this
![A](https://user-images.githubusercontent.com/24843208/204890137-3fd2fc55-19b0-474e-aabf-1dc14f4b5a8d.jpg)

**Comment by yonggamer** [comment]
#105

**Comment by IzaacJ** [comment]
Yes, please. I started to use WSA on my 2-in-1 laptop as a temporary replacement for my failing iPad as my main media consumption device, and this, together with #308, would make WSA a great experience, at least in my usecase :)

**Comment by PourrezJ** [comment]
This would be a very good option to use WSA in a case of HTPC

**Comment by mikaelhug** [comment]
+1, would love a --start-fullscreen option, similar to Google Chromes PWAs.

**Comment by mod-it-max** [comment]
Same

I use my Surface as a tablet often and it's inconvenient to have to pull out a keyboard to fullscreen my apps (drawing apps, games, etc.)
